src/index.json

```diff
@@ -371,15 +384,15 @@
                     "run_requires": [
                         {
                             "requires": [
-                                "python-dateutil (==2.7.3)",
-                                "vsts (==0.1.20)"
```

Comment:
Is there a reason you removed this dependency?

Reply:
This (vsts) is our python client, we have collapsed this into the extension code itself with some optimizations.